modules/text2vec-huggingface/clients/huggingface.go

```diff
@@ -152,7 +152,7 @@ func (v *vectorizer) vectorize(ctx context.Context, url string,
 		return nil, errors.Wrap(err, "cannot decode vector")
 	}
 
-	return &modulecomponents.VectorizationResult{
+	return &modulecomponents.VectorizationResult[[]float32]{
```

Comment:
instead of putting this manually everywhere, is there a way to include it once in the module and then instantiate the client with the proper type?

Reply:
unfortunately in this case not, this is how generics work the type needs to be known for `Vectorize` and `VectorizeQuery` methods - so only 2 places where you need to decide which type of embeddings your vectorizer supports, all other settings are abstracted

Reply:
Maybe it feels like there's a lot of work bc I had to adjust all of the modules, but in fact there are only 2 methods where you need to define the type